test/PositionManager.ts

```diff
@@ -86,6 +88,64 @@ describe("GammaPoolFactory", function () {
         gammaPool = await GammaPool.attach(
             gammaPoolAddr // The deployed contract address
         );
+
+        const createLoanResponse = await (await posMgr.createLoan(cfmm.address, 1, owner.address, ethers.constants.MaxUint256)).wait();
```

Comment:
Don't create the loan here. Create it in the createLoan test section

Reply:
I have created loan here because the tokenId received from it is required throughout other testcases. Passing hardcoded tokenId to other testcases is not working.

Reply:
how does it not work? You shouldn't need to pass an actual tokenId. You can just pass 1 as tokenId if you want. None of the test cases need an actual tokenId